src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -625,6 +626,17 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string? ba
                             }
                             continue;
 
+                        case "generatedartifactsout":
+                            if (string.IsNullOrWhiteSpace(value))
+                            {
+                                AddDiagnostic(diagnostics, ErrorCode.ERR_SwitchNeedsString, MessageID.IDS_Text.Localize(), arg);
+                            }
+                            else
+                            {
+                                generatedArtifactsOutputDirectory = ParseGenericPathToFile(value, diagnostics, baseDirectory);
+                            }
+                            continue;
```

Comment:
as per meeting, there is a dedicated arg for this generatoin.

Reply:
FYI: Also need to update the resx string for `/help` for the compiler, similar to: https://github.com/dotnet/roslyn/blob/0845e86214cf3b084e17e9fcbdbd49ca6f804ca1/src/Compilers/CSharp/Portable/CSharpResources.resx#L4777-L4778

Reply:
Ditto on the help. (We don't have everything but I'd rather we doc stuff as we add it, rather than continuing to get more out of sync).

Reply:
done.